base/applications/mspaint/mouse.cpp

```diff
@@ -162,6 +163,7 @@ struct FreeSelTool : ToolBase
             selectionModel.PushToPtStack(pt);
             imageModel.NotifyImageChanged();
         }
+        updateLast(x, y);
```

Comment:
There seem to be many places where this call is being made. Is it possible to put it in some very specific helper functions so that it does the correct action and doesn't need to be copy-pased in that many functions?

Reply:
- Commonized code.
- Deleted `updateStartAndLast` and `updateLast`.